packages/admin/admin-date-time/src/DatePickerNavigation.tsx

```diff
@@ -1,6 +1,6 @@
-import { createComponentSlot, ThemedComponentBaseProps } from "@comet/admin";
+import { Button, createComponentSlot, ThemedComponentBaseProps } from "@comet/admin";
 import { ArrowLeft, ArrowRight, ChevronDown } from "@comet/admin-icons";
-import { Box, Button, buttonClasses, ComponentsOverrides, IconButton, Menu, menuClasses, MenuItem } from "@mui/material";
```

Comment:
Please adjust the props/variant, as `primary` is the default variant with the Comet Button, unlike with the previous MUI Button. 

<img width="485" alt="DatePicker Story" src="https://github.com/user-attachments/assets/74554f44-397f-4571-9949-f5dac6e1e779" />